boardgame-ui/src/components/activities/ActivityBox.vue

```diff
@@ -0,0 +1,65 @@
+<template>
+    <div v-if=activity class="activity-container">
+        <article class="media" v-if="activity.type === 'event/join'">
+            <div class="media-left calendar-media-left">
+                <figure class="is-64x64"><span><i class="fa-3x far fa-calendar-alt"></i></span></figure>
+            </div>
+            <div class="media-content">
+                {{$t("activity.event.join")}} <strong>{{activity.event.name}}</strong>.
+                <br/>
+                <p class="activity-datetime">{{activity.datetime}}</p>
+            </div>
+        </article>
+        <article class="media" v-if="activity.type === 'game/play'">
+            <div class="media-left">
+                <figure class="image is-64x64"><img class="is-rounded" :src=activity.board_game.thumbnail /></figure>
+            </div>
+            <div class="media-content">
+                {{$t("activity.game.played")}} <strong>{{activity.board_game.name}}</strong>.
+                <br/>
+                <p class="activity-datetime">{{activity.datetime}}</p>
+            </div>
+        </article>
+        <article class="media" v-if="activity.type === 'library/add'">
+            <div class="media-left">
+                <figure class="image is-64x64"><img class="is-rounded" :src=activity.board_game.thumbnail /></figure>
+            </div>
+            <div class="media-content">
+                {{$t("activity.library.added")}} <strong>{{activity.board_game.name}}</strong> {{$t("activity.library.library")}}.
```

Comment:
Better to use named formatting  or list formatting (https://kazupon.github.io/vue-i18n/guide/formatting.html#named-formatting) offered by i18n rather than inserting variables this way.
```suggestion
                {{$t("activity.library.added_game", {gameName: activity.board_game.name})}}.
```
See https://developer.mozilla.org/en-US/docs/Mozilla/Localization/Localization_content_best_practices#Word_Order

If you want a particular style, you can also use https://kazupon.github.io/vue-i18n/guide/interpolation.html#basic-usage
```suggestion
                <i18n path="activity.library.added_game" tag="p">
                  <strong place="gameName">{{ activity.board_game.name }}</strong>
                </i18n>
```